tests/v2/test_0404-array-validity-check.py

```diff
@@ -0,0 +1,861 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/master/LICENSE
+
+from __future__ import absolute_import
+
+import pytest
+import numpy as np
+import awkward as ak
+
+from awkward._v2.tmp_for_testing import v1_to_v2
+
+pytestmark = pytest.mark.skipif(
+    ak._util.py27, reason="No Python 2.7 support in Awkward 2.x"
+)
+
+
+def test_BitMaskedArray():
+    content = ak.layout.NumpyArray(np.arange(13))
+    mask = ak.layout.IndexU8(np.array([58, 59], dtype=np.uint8))
+    array = ak.layout.BitMaskedArray(
+        mask, content, valid_when=False, length=13, lsb_order=False
+    )
+    array = v1_to_v2(array)
+    # FIXME: 'BitMaskedArray' object has no attribute 'bytemask'
+    # assert np.asarray(array.bytemask()).tolist() == [
+    #     0,
+    #     0,
+    #     1,
+    #     1,
+    #     1,
+    #     0,
+    #     1,
+    #     0,
+    #     0,
+    #     0,
+    #     1,
+    #     1,
+    #     1,
+    # ]
```

Comment:
`BitMaskedArray.bytemask` is now named `mask_as_bool` and takes a `validity_error` argument.

Reply:
done